GG/src/Font.cpp

```diff
@@ -1580,13 +1580,13 @@ void Font::FillTemplatedText(
         std::shared_ptr<TextElement> elem = (*te_it);
 
         // For each character in the TextElement.
-        auto it = elem->text.begin();
+        auto text_it = elem->text.begin();
```

Comment:
is this change necessary? not that it's a problem, I just don't see where else in this scope that `it` is used...

Reply:
Chose to replace the shadowed var, the shadowing case is on line 1591 `auto it = m_glyphs.find(c);`